### ISSUE TITLE
all tests are failing because of "Attach server listening on localhost:-1" 

### ISSUE THREAD

**Issue by maplain** [vmware/vic#4789]
time="2017-04-13T21:06:26Z" level=info msg="Attach server listening on localhost:-1" 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x999f4d]

goroutine 24 [running]:
github.com/vmware/vic/lib/portlayer/attach.TestAttachStartStop.func1()
	/drone/src/github.com/vmware/vic/lib/portlayer/attach/server_test.go:48 +0x16d
created by github.com/vmware/vic/lib/portlayer/attach.TestAttachStartStop
	/drone/src/github.com/vmware/vic/lib/portlayer/attach/server_test.go:68 +0x189
FAIL	github.com/vmware/vic/lib/portlayer/attach	0.012s
Makefile:261: recipe for target 'test-job-github.com/vmware/vic/lib' failed
make: *** [test-job-github.com/vmware/vic/lib] Error 1
[info] build failed (exit code 2)

**Comment by jakedsouza** [comment]
The in  is from https://github.com/vmware/vic/blob/master/lib/portlayer/attach/server.go#L56

Which occurs due to 

https://golang.org/src/net/lookup.go
```
func (r *Resolver) LookupPort(ctx context.Context, network, service string) (port int, err error) {
   223		port, needsLookup := parsePort(service)
   224		if needsLookup {
   225			port, err = r.lookupPort(ctx, network, service)
   226			if err != nil {
   227				return 0, err
   228			}
   229		}
   230		if 0 > port || port > 65535 {
   231			return 0, &AddrError{Err: "invalid port", Addr: service}
   232		}
   233		return port, nil
   234	}
```

**Comment by anchal-agrawal** [comment]
Fixed in master by #4792 and in release by #4793.
